js/sandbox.js

```diff
@@ -33,7 +33,7 @@ var Sandbox =  {
                 }         
             },
             listen : function (evts) {
-                if (core.is_obj(evts)) {
+                if (core.is_obj(evts) || core.is_arr(evts)) {
```

Comment:
We always modify core when we need to add/change the implementation of stuff. Then we modify sandbox which is a facade between the modules and core.